index.html

```diff
@@ -26,20 +26,42 @@
   <body>
     <header>
       <nav>
-        <p class="title">Mumenya Nyamu</p>
+        <p class="title" id="title">Mumenya Nyamu</p>
         <div class="open-toggle" id="open-toggle">
           <i class="fa fa-bars" aria-hidden="true"></i>
         </div>
-        <ul class="link">
-          <li><a class="is-link" href="#">Portfolio</a></li>
-          <li><a class="is-link" href="#">About</a></li>
-          <li><a class="is-link" href="#">Contact</a></li>
+        <ul class="link main-links" id="main-links">
+          <li><a class="is-link link-color" href="#">Portfolio</a></li>
+          <li><a class="is-link link-color" href="#">About</a></li>
+          <li><a class="is-link link-color" href="#">Contact</a></li>
```

Comment:
- Optional
When I click on the navbar links in the desktop mode, they don't display anything, instead, it would be better if I click say on the portfolio link and it takes me to the portfolio section. Also, the animation on the navbar elements is not looking good. You can consider removing it or changing it.